src/Network/Receive.pm

```diff
@@ -1427,44 +1427,57 @@ sub sage_autospell {
 
 sub show_eq {
 	my ($self, $args) = @_;
-
-	my $jump = 26;
-
-	my $unpack_string  = "v ";
-	   $unpack_string .= "v C2 v v C2 ";
-	   $unpack_string .= "a8 ";
-	   $unpack_string .= "a6"; #unimplemented in eA atm
-
-	if (exists $args->{robe}) {  # check packet version
-		$unpack_string .= "v "; # ??
-		$jump += 2;
+	my $item_info;
+	my @item;
+	
+	if ($args->{switch} eq '02D7') {  # PACKETVER DEFAULT	
+		$item_info = {
+			len => 26,
```

Comment:
Many implementations of the other servertypes will often override the packing string (because the server sends different stuff). This way, the solution is more generic and will support all server types.